### ISSUE TITLE
Problems/Questions about Crypto Wallet

### ISSUE THREAD

**Issue by makkiato83** [brave/brave-browser#7753]
Hello, I originally posted in Reddit:

https://www.reddit.com/r/brave_browser/comments/epi29k/problemsquestions_about_crypto_wallet/ 

and was asked to re-write here for better visibility

I would like to report a couple of problems with Brave crypto wallet (metamask fork) I have encountered (Version 1.2.42 Chromium: 79.0.3945.117 (Official Build) (64-bit), MacOSX).

1. If in Settings/Extensions "Crypto Wallet" is disabled, clicking on Settings/Crypto Wallet (top right) (i.e., going to chrome://wallet/) gives a bad looking message:
"wallet is blocked, Requests to the server have been blocked by an extension, Try disabling your extensions. ERR_BLOCKED_BY_CLIENT".
I thing it would be better to write a better-looking message like "You have currently disabled the Crypto wallet in your Settings! Go to "chrome://settings/extensions" to activate it"

2. After activating the wallet in Settings/Extensions, I created an account. However i forgot the password (and seed phrase, it was just an experiment). Now I don't see how to create another new account? The Crypto wallet only offers me the possibility of recovering the account with the seed phrase. I could not reset the Brave Crypto wallet simply by de/activating it in Settings.

3. Is it possible to manage multiple accounts in the Brave Crypto wallet? For example, one associated with a Trezor, another different one with a Ledger and and a third hot-wallet?

4. There is a conflict between Brave Crypto Wallet and Metamask, the two cannot be used together. Can this be fixed?

Thanks!

**Comment by srirambv** [comment]
1 and  4 will be fixed with https://github.com/brave/brave-browser/issues/7503. 

2 will be fixed with https://github.com/brave/brave-browser/issues/5741 allowing to reset account

As for 3 you can add multiple accounts and connect hardware keys but having multiple accounts and restoring it with seed keys will only restore the first account. This is a known limitation from [upstream ](https://github.com/MetaMask/metamask-extension/issues/5065)

cc: @bbondy @ryanml

**Comment by makkiato83** [comment]
Thanks @srirambv for the very quick reply!

Regarding 3, how can I add accounts? I don't see this option, currently.

What I see is: https://www.dropbox.com/s/cscemvkurvzxg5s/Screen%20Shot%202020-01-16%20at%2020.41.09.png

**Comment by bbondy** [comment]
Re 1&4: This is fixed on Nightly and Dev where something landed to improve compatibility of Brave and MetaMask.
Re 3: Yes I think it should be possible already.
Re 2: There's only 1 seed but you can create multiple wallets from that seed. There will be a reset wallet functionality coming within the next few versions.

Closing this as I think all things are answered. Let me know if you have further questions

**Comment by Unapapawu** [comment]
This is a great article with lots of fantastic writeups, especially the room for meaningful discussions. I've also read a lot of stories about people losing money to investment scams. I too have been a victim of this scams. I lost about $50k Bitcoin last year, I searched around and tried to work with people ,unfortunately I was scammed as well. This happened for months until I came across (Easyreclaiming @ gmail z com) they came to my rescue and all my funds were recovered.
